src/components/super-networked-interactable.js

```diff
@@ -82,7 +88,15 @@ AFRAME.registerComponent("super-networked-interactable", {
     this.el.setAttribute("body", { type: "static" });
     this.el.emit("grab-end", { hand: this.hand });
     this.hand = null;
-    this.counter.deregister(this.el);
+    this._syncCounterRegistration();
+  },
+
+  _onPinned: function() {
```

Comment:
I would ditch these 2 methods and just bind `_syncCounterRegistration` to the events directly